### PR TITLE
C(WL) gull

### PR DESCRIPTION
**Pull request by mcovarr**
A proposed config scheme to support multiple language frontends to Cromwell.  Basically a simplified version of the backend scheme, this is just a strawman for discussion.

### CODE REVIEW

File: core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -209,6 +209,37 @@ engine {
...
 }
 
+frontend {
+  default = "WDL"
+  providers {
+    WDL {
+      supported-versions = [
+        {
+          workflow-type = [ "WDL" ],
+          workflow-type-version = [ ] # actually require this to be blank as assent to Cromwell's disregard for WDL versions?
```
Comment (geoffjentry) [inline core/src/main/resources/reference.conf]: We should just assign some arbitrary version to pre-version WDL, e.g. `draft-2`, which is technically what we say it is. After that if we make clear that draft versions are moving targets I think it'll be fine.

File: core/src/main/resources/reference.conf [hunk 2]
```diff
@@ -209,6 +209,37 @@ engine {
   }
 }
 
+frontend {
+  default = "WDL"
+  providers {
+    WDL {
+      supported-versions = [
+        {
+          workflow-type = [ "WDL" ],
```
Comment (geoffjentry) [inline core/src/main/resources/reference.conf]: I don't understand why this is an array

**Comment by geoffjentry** [comment]
I thought @cjllanwarne brought up a good point at standup about whether or not we wanted this to be pluggable in the first place. I didn't actually think about that when I looked at this. I suspect that usage of actual plugability would be pretty low.

**Comment by mcovarr** [comment]
I'm going to table this for now as there's more interesting stuff to be working on.

**Pull request closed without merging**